subprojects/logging/src/main/java/org/gradle/internal/logging/events/ProgressCompleteEvent.java

```diff
@@ -16,18 +16,21 @@
 
 package org.gradle.internal.logging.events;
 
+import org.gradle.api.Nullable;
 import org.gradle.api.logging.LogLevel;
 
 public class ProgressCompleteEvent extends CategorisedOutputEvent {
     private final String status;
     private final String description;
-    private OperationIdentifier operationId;
+    private OperationIdentifier progressOperationId;
+    private final Object buildOperationId;
```

Comment:
we don't need build operation id on this kind of event, as we know the build operation id given the progress operation id (on the start event).

The rename of `operationId` is good though.